client/src/components/Reviews/components/Ratings.jsx

```diff
@@ -6,13 +6,52 @@ const Container = styled.div`
   border: 2px solid black;
   margin: 10px;
 `;
+const Bar = styled.div`
+  background-color: #a6a6a6;
+  height: 5px;
+  margin: 2px;
+  border-radius: 20px;
+`;
+const StyledDiv = styled.div`
+  padding-right: ${(props) => props.stars}%;
+`;
+const AllRatings = styled.div`
+  text-align: left;
+`;
 
 function Ratings({ reviewsMetadata }) {
-  // TODO parse the data that we get passed in to render things
+  let total = 0;
+  let totalStars = 0;
+  let highest = 0;
+  Object.values(reviewsMetadata.ratings).forEach((num) => { total += num; });
```

Comment:
When you want to compute a single number from an array, `.reduce()` is usually what you want to reach for:

`Object.values(reviewsMetadata.ratings).reduce((numberOfRating, total) => numberOfRating + total, 0)`